.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [18.x, 20.x, 21.x]
+        node-version: [16.x, 18.x, 20.x, 22.x]
```

Comment:
Node 16 had its security support end 8 months ago, how long do you plan to support it?